### PR TITLE
fix(account): fix the ability to delete the current account from sidebar

### PR DESCRIPTION
**Pull request by tamazlykar**
This is a fix for the [previous PR](https://github.com/bwsw/cloudstack-ui/pull/1043) that was merged by mistake

### CODE REVIEW

**Comment by tamazlykar** [comment]
@vanekbr changes provided

File: src/app/shared/utils/account.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { User, Account } from '../models';
+
+export function isHaveSameAccount(user: User, account: Account) {
+  return user.account === account.user[0].account;
```
Comment (zolotyx) [inline src/app/shared/utils/account.ts]: Please make sure that user array is not empty

Comment (tamazlykar) [reply src/app/shared/utils/account.ts]: API doc tells that account.user implies element has a child.
Do I need to write an additional check in the code?

Comment (zolotyx) [reply src/app/shared/utils/account.ts]: yes, and please, rename function to represent the meaning "User belongs to the account"

**Pull request merged**